pkgs/top-level/go-packages.nix

```diff
@@ -1230,6 +1230,30 @@ let
     };
   };
 
+  go-runewidth = buildGoPackage rec {
+    rev = "travisish";
+    name = "go-runewidth-${rev}";
+    goPackagePath = "github.com/junegunn/go-runewidth";
+    src = fetchFromGitHub {
+      inherit rev;
+      owner = "junegunn";
+      repo = "go-runewidth";
+      sha256 = "07d612val59sibqly5d6znfkp4h4gjd77783jxvmiq6h2fwb964k";
+    };
+  };
+
+  go-shellwords = buildGoPackage rec {
+    rev = "35d512af75e283aae4ca1fc3d44b159ed66189a4";
+    name = "go-shellwords-${rev}";
+    goPackagePath = "github.com/junegunn/go-shellwords";
```

Comment:
This is actually forked from https://github.com/mattn/go-shellwords, so maybe I should put the junegunn version under a different name?
Though the original has been last updated 2 years ago...